dpsim-models/include/dpsim-models/Definitions.h

```diff
@@ -78,6 +78,14 @@ namespace CPS {
 	typedef Eigen::Matrix<Real, Eigen::Dynamic, Eigen::Dynamic, Eigen::ColMajor> Matrix;
 	/// @brief Dense matrix for complex numbers.
 	typedef Eigen::Matrix<Complex, Eigen::Dynamic, Eigen::Dynamic, Eigen::ColMajor> MatrixComp;
+	/// @brief Dense 2-by-2 matrix for real numbers.
```

Comment:
Cant we use a template here? There is really no point in defining types for each size of matrix.. 

Templates can be partially specialized leaving only the size to choose by the user.

Reply:
I agree. Using an [alias template](https://en.cppreference.com/w/cpp/language/type_alias), like for [MatrixVar](https://github.com/sogno-platform/dpsim/blob/450848733cd77229b577b91da11dab5664b5475d/dpsim-models/include/dpsim-models/Definitions.h#L100-L101), seems reasonable. I would propose then to introduce `MatrixFixedSize<int Rows, int Cols>` for `Real` numbers and `MatrixFixedSizeComp<int Rows, int Cols>` for `Complex` numbers

Reply:
This is done now. Custom definitions are removed and `MatrixFixedSize<int rows, int cols>` is defined (also complex case) and in use